native/source/mixpanel/detail/worker.hpp

```diff
@@ -41,11 +51,16 @@ namespace mixpanel
                 Result send_engage_batch();
                 Result send_batch(const std::string& name, bool verbose);
 
+                int parse_www_retry_after(const nanowww::Response& response);
+                int calculate_back_off_time(int failure_count);
```

Comment:
Add `static` in front of this line, since this function doesn't need access to `*this`.